Model/Page.php

```diff
@@ -1,5 +1,27 @@
 <?php
 
-class Training_Cms_Model_Page extends Mage_Core_Model_Abstract{
+class Training_Cms_Model_Page extends Mage_Core_Model_Abstract
+{
+
+    /**
+     * init model
+     *
+     */
+    protected function _construct()
+    {
+        $this->_init('training_cms/page');
+    }
+
+    /**
+     * Loading page by code
+     *
+     * @param $code
```

Comment:
If you are adding parameter declaration in PHPDoc block because there is no type hint for the parameter in method's declaration, then add parameter's type to PHPDoc block.